q2.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```

Comment:
Change filename `q1.html` to `index.html`

Reply:
Okay Sir